### PR TITLE
Added support for radio channel groups.

### PR DESCRIPTION
**Pull request by ksooo**
Added support for radio channel groups.

Please note that this feature relies on a Kodi fix. https://github.com/xbmc/xbmc/pull/6755

### CODE REVIEW

**Comment by ksooo** [comment]
BTW: why don't we have a "feature" label?

**Comment by xhaggi** [comment]
because these are the default ;)

**Comment by ksooo** [comment]
hehe.

**Comment by xhaggi** [comment]
i've change the defaults (replace 'enhancement' with 'Improvement') because IMO this is an improvement

File: src/Tvheadend.cpp [hunk 1]
```diff
@@ -113,7 +113,23 @@ int CTvheadend::GetTagCount ( void )
   return m_tags.size();
 }
 
-PVR_ERROR CTvheadend::GetTags ( ADDON_HANDLE handle )
+bool CTvheadend::IsTagKindMatch( const STag &tag, bool bRadio ) const
```
Comment (xhaggi) [inline src/Tvheadend.cpp]: `IsTagContainChannelType(const STag &tag, bool bRadio)`?

**Comment by xhaggi** [comment]
apart from my comment looks good ;)

**Comment by ksooo** [comment]
@xhaggi IsTagKindMatch => IsTagContainsChannelType now. Much better now, imo.

**Comment by xhaggi** [comment]
sounds good now ;)

**Comment by Jalle19** [comment]
I would drop the "Is" from the method name.

**Comment by ksooo** [comment]
@Jalle19 done. Dropped the "Is". Anything else?

**Comment by Jalle19** [comment]
Looks good. To me it would make more sense to change some of our structs into classes and add some methods to them, but that's fof another time.

**Comment by ksooo** [comment]
I agree. Let's do this over time whenever it makes sense. I thought of this yesterday while adding the new method to Tvheadend class. But adding this new method to STag makes not much sense to me, as implementation needs access to a Tvheadend member. But on second thought... Will rethink.

**Comment by ksooo** [comment]
@Jalle19 couldn't resist. STag is a class now. :-) Please review.

**Comment by ksooo** [comment]
class STag? "S" stands for struct, right? wtf! Will change this to namespace htsp { class Tag ... } later today.

**Comment by Jalle19** [comment]
Yeah the S is for structs only.

**Comment by ksooo** [comment]
Corrected the naming for the new class now.
@Jalle19 mind taking a look?

**Comment by ksooo** [comment]
If nobody objects I will merge this later tonight. Version bump will follow later.

**Comment by xhaggi** [comment]
+1

**Pull request merged**